keras/ops/nn_test.py

```diff
@@ -1750,6 +1750,22 @@ def test_batch_normalization(self):
         )
         self.assertEqual(tuple(output.shape), (2, 3, 3, 5))
 
+    def test_ctc_loss(self):
```

Comment:
Please also add a shape inference test (calling the function on KerasTensors and checking the shape/dtype of the output) -- see the TestCase class above

Reply:
Done, thank you!